NetworkingServiceKit.podspec

```diff
@@ -38,4 +38,5 @@ NetworkingServiceKit is the reincarnation of the standard iOS monolith api clien
     s.dependency 'AlamofireImage'
     s.dependency 'CryptoSwift'
     s.dependency 'SwiftyJSON'
+    s.dependency 'ReactiveSwift'
```

Comment:
I would argue that this integration should be in a subspec. People should be able to use NSK without opting into ReactiveSwift.  This also would give us the opportunity to create extensions for other popular FRP frameworks like RxSwift